src/core/webview/ClineProvider.ts

```diff
@@ -1943,7 +1943,12 @@ export class ClineProvider
 	 * like the current mode, API provider, git repository information, etc.
 	 */
 	public async getTelemetryProperties(): Promise<TelemetryProperties> {
-		const { mode, apiConfiguration, language } = await this.getState()
+		const {
+			mode,
+			apiConfiguration,
+			language,
+			experiments, // kilocode_change
+		} = await this.getState()
```

Comment:
#nit not sure if we like this kinda thing, up to you!

```suggestion
		const { mode, apiConfiguration, language } = await this.getState()
		const { experiments } = await this.getState() // kilocode_change
```

Reply:
not sure `getState()` doesn't look super cheap (includes file io for example)